src/color.cpp

```diff
@@ -654,7 +654,7 @@ std::string colorize( const translation &text, const nc_color &color )
 
 std::string get_note_string_from_color( const nc_color &color )
 {
-    for( auto i : color_by_string_map ) {
+    for( const auto &i : color_by_string_map ) {
```

Comment:
Eschew auto
```suggestion
    for( const std::pair<std::string, note_color> &i : color_by_string_map ) {
```

Reply:
Should probably be
```suggestion
    for( const std::pair<const std::string, note_color> &i : color_by_string_map ) {
```
to make clang 10 happy.